go-gorilla/src/internal/routes/routes.go

```diff
@@ -63,6 +64,13 @@ func SetRouteHandlers(r *mux.Router) {
 	r.HandleFunc("/{username}/follow", handlers.Follow_user)
 	r.HandleFunc("/{username}", handlers.User_timeline)
 	r.HandleFunc("/{username}/unfollow", handlers.Unfollow_user)
+
+	//API
+	r.HandleFunc("/api/msgs", handlers.API_Messages).Methods("GET").Name("Messages")
```

Comment:
It would be more clean if you had two functions such as SetRouteHandlersUI and SetRouteHandlersAPI but it is totally fine like this. It would work the same. Not a necessary change.